google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/main/java/com/google/cloud/spanner/jdbc/JdbcDataType.java

```diff
@@ -0,0 +1,267 @@
+/*
```

Comment:
I think this was already done in step 13. Does it need a rebase?

Reply:
I based this PR on step 13 to avoid build errors. Once 13 is merged, this one should be rebased.

Reply:
It has now been rebased.